numpy/testing/_private/extbuild.py

```diff
@@ -240,11 +229,15 @@ def build(cfile, outputfilename, compile_extra, link_extra,
                               cwd=build_dir,
                               )
     else:
-        subprocess.check_call(["meson", "setup", "--vsenv", ".."],
```

Comment:
Not sure about this one. That's going to prefer any old or broken compiler over MSVC (e.g., the MinGW one shipped by GHA) when multiple compilers are installed on a machine. Is this necessary?

Reply:
`--vsenv` should be safe on all platforms, it's a no-op on non-win32. Maybe it landed in an `else:` block later though - I agree it shouldn't matter then.